ksql-engine/src/main/java/io/confluent/ksql/function/InternalFunctionRegistry.java

```diff
@@ -58,8 +59,8 @@ public InternalFunctionRegistry() {
   }
 
   private InternalFunctionRegistry(final Map<String, UdfFactory> ksqlFunctionMap,
-                                   final Map<String, AggregateFunctionFactory>
-                                       aggregateFunctionMap) {
+      final Map<String, AggregateFunctionFactory>
```

Comment:
nit: whitespace changes. The params should line up

Reply:
Now it's one parameter per line.